### ISSUE TITLE
LinuxMint Cinnamon - No way to quit Settings - Missing window bar with close "X" button - Use Alt+F4

### ISSUE THREAD

**Issue by mikkovedru** [olivierkes/manuskript#464]
Loading settings works fine, but there is no way to quit them:
- There is no button to click, that would close the Settings.
- Esc button does nothing.
- Even pressing F8 again does nothing.

The only way to quit is to close the whole program and start it again.

All three of the mentioned ways should work - at the very least pressing Esc and having a button.

**Comment by gedakc** [comment]
What Operating System (OS) and screen resolution are you using, and which version of Manuskript?

The Settings window should have the normal **[Minimize]**, **[Maximize]**, and **[Close]** buttons that are standard with the OS.

The following screen shot from Kubuntu 16.04 LTS shows these buttons in the upper right hand corner of the window.  The **[Close]** button is the "**X**".

![manuskript-settings](https://user-images.githubusercontent.com/10405019/52069738-93d53600-253c-11e9-8acd-fbfb354af79f.png)

Additionally many OSes include a keyboard shortcut to close a window.  On Kubuntu the key combination is **Alt+F4** to close a window.

**Comment by mikkovedru** [comment]
Linux Mint 19.1 Cinnamon.
![image](https://user-images.githubusercontent.com/2521942/52070574-d07b4e80-2589-11e9-9617-7bf43dd39d67.png)

![image](https://user-images.githubusercontent.com/2521942/52070599-de30d400-2589-11e9-9701-c974ae318f9b.png)

**Comment by gedakc** [comment]
The screen shot definitely shows that the normal window buttons are missing.  I've never used the Cinnamon window manager before.  I suspect there is some incompatibility between it and Manuskript.

Does the main Manuskript window have a **[Close]** button?

Does the keyboard shortcut **Alt+F4** close the Settings window?

**Comment by mikkovedru** [comment]
> Does the main Manuskript window have a [Close] button?
Yes. It has all the main buttons and they all work totally fine.

> Does the keyboard shortcut Alt+F4 close the Settings window?
Yes.

**Comment by gedakc** [comment]
From some Internet searches it appears that the Window decorations (I assume this includes the window bar, title, and buttons) can be disabled in Cinnamon.  See [Linux Mint Cinnamon - GUI option to remove window decorations?](https://github.com/linuxmint/Cinnamon/issues/3217#issuecomment-297993973)

Can you check to see if you have such an option and if so try enabling Window decorations?

**Comment by mikkovedru** [comment]
> From some Internet searches it appears that the Window decorations (I assume this includes the window bar, title, and buttons) can be disabled in Cinnamon. See [Linux Mint Cinnamon - GUI option to remove window decorations?](https://github.com/linuxmint/Cinnamon/issues/3217#issuecomment-297993973)
> 
> Can you check to see if you have such an option and if so try enabling Window decorations?

I use only 2 extensions: Coverflow Alt-Tab and Desktop Scroller (switch between desktop by scrolling at the edges of the screen).

The other link is very old and not applicable to current Cinnamon.

Here are my Windows settings - I don't think there is anything weird in here.
![image](https://user-images.githubusercontent.com/2521942/52074837-72ebff80-2593-11e9-9867-ee88ac56a5e3.png)
![image](https://user-images.githubusercontent.com/2521942/52074557-d4f83500-2592-11e9-8bb8-317172e9ae16.png)
![image](https://user-images.githubusercontent.com/2521942/52074568-d9245280-2592-11e9-90cb-92b9e7baf511.png)

Now that I think of it, I do remember encountering the same kind of problem (no closing buttons for the internal window) with at least one other program. At that time I though of it as a typical Linux weirdness (and assumed either a bug or bad UI/product skills of the programmers), but didn't report it. Too bad I can't remember what program that was. :-(

**Comment by gedakc** [comment]
I just tested Linux Mint 19 Cinnamon in a Virtual Machine and the Settings window does have a close "**X**" button as shown in the following screen shot.

![linuxmint19cinnamon-settings-with-close-button](https://user-images.githubusercontent.com/10405019/52081207-e9b6d780-2556-11e9-8251-43a7e9fc0a61.png)

Hence the issue does appear to be specific to the configuration on your computer.

I will update the title of this issue and label it as a question.  That way others can more easily find this issue.

**Comment by mikkovedru** [comment]
Thank you for spending time investigating this issue!

You installed version 19, while I was having problems with the version 19.1...

But I did download version 19.1 and installed it in my VM. And yeah, Manuskript works in version 19.1 as well.

So I started to investigate, started comparing all the system settings and changing them. And I found the problem!

The problem was located in System Settings / Themes. There are a bunch of settings for the style of OS ("Window borders", "Icons", "Controls", etc). Since I have been having my system from 2014, and always upgrading to a newer version (never a clean install), there were apparently some residues (I think it was the theme with increased contrast in the "Window borders" sub-setting) that caused this issue. Changing it to a new one completely solved the issue.

Thank you for being a great help!

I hope this topic will help other people with the same problem.
